legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/api/GeneratorResultAbstract.java

```diff
@@ -38,7 +38,7 @@
 )
 public abstract class GeneratorResultAbstract
 {
-    private static final String SINGLE_QUOTE = "'";
+    public static final String SINGLE_QUOTE = "'";
```

Comment:
Seems odd to define a general-purpose constant on this class.  Even further, do we need a constant for this well know value?

Reply:
Will find usage and cleanup in forthcoming PR